unattended_installer/cert_tool/certFunctions.sh

```diff
@@ -23,13 +21,40 @@ function cert_checkOpenSSL() {
     fi
 }
 
+function cert_checkRootCA() {
```

Comment:
Please follow the script convention and add an empty line after the function declaration.

Reply:
Done